apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue

```diff
@@ -186,7 +191,7 @@ function getSettingValue(settingKey: string, settings: ISetting[]) {
             "CATALOGUE_LANDING_SAMPLES_TEXT",
             data.data._settings
           ) ||
-          "The cumulative number of participants with samples collected of all datasets combined."
+          "The cumulative number of participants with samples collected of all datasets combined"
```

Comment:
so we assume subcohorts are not overlapping?

Reply:
i guess so, the maintainer can override the default/fallback text